src/nested-list.scss

```diff
@@ -0,0 +1,295 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+$block: #{$fd-namespace}-nested-list;
+
+@mixin fd-side-nav-arrow {
+  font-family: "SAP-icons";
+  font-style: normal;
+  font-weight: normal;
+  color: var(--sapList_TextColor);
+  font-size: var(--sapFontHeader4Size);
+  text-decoration: inherit;
+  text-transform: none;
+  speak: none;
+}
+
+@mixin fd-flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.#{$block} {
+  @include fd-reset();
+
+  list-style: none;
+  width: 100%;
+
+  @for $i from 2 through 10 {
+    .level-#{$i} {
+      $paddingLeft: ($i - 2) + 2.75rem;
+
+      .#{$block}__link,
+      .#{$block}__content {
+        padding-left: $paddingLeft;
```

Comment:
As @bcullman mentioned previously, I don't think we want to be determining the "level" of nesting and calculating padding for each... wouldn't it be preferred to instead put padding on each "parent" `ul` (with some modifier like `.fd-list--expandable`) and let the children indent themselves (recursively)? It would eliminate the need for this custom math.

As a visible example from UI5 wiki:
<img width="420" alt="Screen Shot 2019-10-22 at 9 31 03 AM" src="https://user-images.githubusercontent.com/3812133/67309066-16799900-f4b0-11e9-9d22-95753df60e1e.png">

https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2075534845

Reply:
Yes, in most of the cases this would be the approach, but not in this one. If you add padding to the ul element then the `selected` styling of the subitems (children) wouldn't be correct. The background color and the underline border should go to the ends of the Navigation list.
Here is how it looks now:
<img width="1491" alt="Screen Shot 2019-10-22 at 1 25 50 PM" src="https://user-images.githubusercontent.com/39598672/67312863-0671b180-f4d0-11e9-9278-fa703d1bae0a.png">

Here is the case when we add padding to the `ul` element and don't do the calculations
<img width="1493" alt="Screen Shot 2019-10-22 at 1 29 20 PM" src="https://user-images.githubusercontent.com/39598672/67312904-17babe00-f4d0-11e9-8971-dd3f9b1241f2.png">

Reply:
Makes sense, I didn't see the hover/active states previously.

Reply:
I still don't know that we want the consumers of this to have to keep track of the "level" of indentation in their markup to supply the correct class.

Seems like using parent/child relationship selectors allows consumers to just apply classes more generically, and the CSS is smart enough to style itself, like:
```css
.list
    .list__item
    .list__item
        .list__item
        .list
            .list__item
            .list__item
        .list__item
    .list__item
```